code/controllers/subsystem/polling.dm

```diff
@@ -0,0 +1,216 @@
+SUBSYSTEM_DEF(polling)
+	name = "Polling"
+	flags = SS_BACKGROUND | SS_NO_INIT
+	wait = 1 SECONDS
+	runlevels = RUNLEVEL_GAME
+	/// List of polls currently ongoing, to be checked on next fire()
+	var/list/datum/candidate_poll/currently_polling
+	/// Whether there are active polls or not
+	var/polls_active = FALSE
+	/// Number of polls performed since the start
+	var/total_polls = 0
+	/// The poll that's closest to finishing
+	var/datum/candidate_poll/next_poll_to_finish
```

Comment:
This doesn't feel like it should be on the heap at all--introduces the possibility of drift. This should be calculated when things request it.